src/ansys/mapdl/core/pool.py

```diff
@@ -367,9 +367,8 @@ def __init__(
             [thread.join() for thread in threads]
 
             # make sure everything is ready
-            timeout = time.time() + timeout
-
-            while timeout > time.time():
+            n_instances_ready = 0
+            while time.time() + timeout > time.time():
```

Comment:
I guess this will always be true

```suggestion
            timeout = time.time() + timeout
            while timeout > time.time():
```

Reply:
You are right!